### ISSUE TITLE
GRAND-Slam on nascentRNA data

### ISSUE THREAD

**Issue by ezecalvo** [erhard-lab/gedi#69]
Hi,

I was wondering if you have any recommendations to run GRAND-Slam in nascentRNA-enriched samples. For example by doing a 4sU-biotin pulldown followed by the IAA treatment like people do on [TT-Timelapse Seq](https://simonlab.yale.edu/research/transcriptome-dynamics/tt-timelapse-seq/).

Thanks!

**Comment by isaacvock** [comment]
This is a very late response, but I thought I would put something out here in case others come across this post wondering what the answer is. 

Modeling of the mutational data in TT-TimeaLapse-seq and similar enrichment base methods is largely unexplored. In theory, a benefit of these methods is that the mutational data can be used to ascertain what fraction of your reads from sequencing the enriched sample are truly labeled reads, and what fraction come from unlabeled RNA "contamination". Thus, you could imagine using GRAND-SLAM or [a similar tool](https://github.com/simonlabcode/bakR) to estimate the new-to-total ratio (NTR; sometimes alternatively referred to as the "fraction new") using the same model that these methods use for enrichment-free approaches. The only problem is that it is unclear the extent to which the simple two-component mixture model (i.e., sequencing reads are a mixture of two distinct populations: new and old reads, with the latter having a higher mutation rate than the former), applies in the context of these enrichment + sequencing experiments. The fact that the probability of enriching a fragment from a labeled RNA is likely a complex function of the number of 4sUs in that fragment, combined with the fact that the short label times used in these experiments mean that the availability of 4sU is probably ramping up during the short feed and thus different newly synthesized RNA fragments are exposed to different 4sU incorporation raets, likely causes data from these experiments to deviate from the simple assumptions made by models designed for enrichment-free experiments. Whether or not these deviations are strong enough to significantly impact the accuracy/usefulness of two-component mixture modeling in this setting is unknown.

In short, one idea would be to multiply the read counts for each gene by the NTR provided by GRAND-SLAM, and use that as an estimate of the "number of 4sU labeled reads", though rigorous tests of this strategy have not been presented in the formal literature as of yet. A simple alternative is to just filter out reads with less than a certain number of mutations (what mutation count cutoff makes sense depends on how well your cells took up 4sU and how long your reads are), and assume that this strategy filters out most of the reads from unlabeled RNA. You could use the "min2" column in the GRAND-SLAM output for this, or an [alternative data processing pipeline](https://github.com/simonlabcode/bam2bakR.git) that outputs mutational content information for all reads.
